pageserver/src/http/routes.rs

```diff
@@ -105,6 +105,9 @@ impl From<PageReconstructError> for ApiError {
             PageReconstructError::Cancelled => {
                 ApiError::InternalServerError(anyhow::anyhow!("request was cancelled"))
             }
+            PageReconstructError::AncestorStopping(_) => {
+                ApiError::InternalServerError(anyhow::anyhow!(pre))
```

Comment:
```suggestion
                ApiError::InternalServerError(anyhow::Error::new(pre))
```

Seems a bit weird otherwise to switch style between arms. Effect might be the same.